*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:36875 (IPv4)
Extension host agent listening on 36875

[16:52:11] Extension host agent started.
[16:52:12] [127.0.0.1][88ae6f47][ManagementConnection] New connection established.
[16:52:12] [127.0.0.1][973b8109][ExtensionHostConnection] New connection established.
[16:52:12] [127.0.0.1][973b8109][ExtensionHostConnection] <5265> Launched Extension Host Process.
[17:07:13] No ptyHost heartbeat after 6 seconds
[17:07:52] No ptyHost heartbeat after 6 seconds
[17:08:24] No ptyHost heartbeat after 6 seconds
[17:11:03] No ptyHost heartbeat after 6 seconds
[17:11:04] [127.0.0.1][88ae6f47][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[17:11:04] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
rejected promise not handled within 1 second: ErrorNoTelemetry: Could not find pty on pty host
stack trace: ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateIcon (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[17:11:05] [ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateIcon (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
rejected promise not handled within 1 second: ErrorNoTelemetry: Could not find pty on pty host
stack trace: ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[17:11:05] [ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
rejected promise not handled within 1 second: ErrorNoTelemetry: Could not find pty on pty host
stack trace: ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[17:11:05] [ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/russellmeng21/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
[17:11:06] [127.0.0.1][973b8109][ExtensionHostConnection] <5265> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[17:11:06] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[17:11:09] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[17:11:09] Cancelling previous shutdown timeout
Got delay-shutdown request while in shutdown timeout, delaying
[17:11:10] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[17:11:10] Cancelling previous shutdown timeout
[17:11:11] [127.0.0.1][88ae6f47][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[17:11:11] [127.0.0.1][ff79e826][ManagementConnection] New connection established.
[17:11:11] [127.0.0.1][94eafb7f][ExtensionHostConnection] New connection established.
[17:11:11] [127.0.0.1][94eafb7f][ExtensionHostConnection] <32157> Launched Extension Host Process.
[IPC Library: Pty Host]  WARN Persistent process "1": Process had no disconnect runners but was an orphan
[IPC Library: Pty Host]  INFO Persistent process "1": Replaying 100 chars and 1 size events
[17:11:17] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/russellmeng21)
New EH opened, aborting shutdown
[17:16:10] New EH opened, aborting shutdown
[17:16:11] [127.0.0.1][88ae6f47][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[IPC Library: Pty Host]  INFO Persistent process "6": The short reconnection grace time of 5m has expired, shutting down pid 5309
[23:05:50] [127.0.0.1][94eafb7f][ExtensionHostConnection] <32157> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[23:05:50] Last EH closed, waiting before shutting down
Last EH closed, shutting down
[23:10:50] Last EH closed, shutting down
